lib/ddtrace/profiling/collectors/stack.rb

```diff
@@ -68,7 +68,7 @@ def collect_events
 
           # Compute wall time interval
           current_wall_time = Datadog::Utils::Time.get_time
-          wall_time_interval_ns = (current_wall_time - last_wall_time) * 1e9
+          wall_time_interval_ns = ((current_wall_time - last_wall_time).round(9) * 1e9).to_i
```

Comment:
Can't do this unfortunately; Ruby 2.0 doesn't support `Process.clock_gettime`, hence why we use `Datadog::Utils::Time.get_time` to abstract this. We could maybe move this unit to `Datadog::Utils::Time#get_time` instead, so we don't have to do this kind of computation.

Reply:
I agree that having an abstraction around it would be good, given we have to sniff for local function support. My suggestions is mostly to reduce the arithmetic operations here. If you think extracting it would improve maintainability, I suggest adopting something along the lines of my suggestion, but I leave it to your judgment.